pkg/standalone/common.go

```diff
@@ -0,0 +1,20 @@
+package standalone
+
+import (
+	"os"
+	path_filepath "path/filepath"
+	"runtime"
+)
+
+// getDefaultComponentsFolder returns the hidden .components folder created under install directory at init time
```

Comment:
nitpick: change the comment: `returns the path to default components folder under home directory at init`

Reply:
fixed